lib/mastermind_new.rb

```diff
@@ -47,13 +47,13 @@ def self.play
     end
 
     class GeneratedList
-        attr_reader :get
+        attr_reader :get_list
```

Comment:
It feels like this could just be turned into a class method like
```class Generated List
  def self.generate
     ALL THAT CODE
```

And then called list = GeneratedList.generate.

Doing this type of work in initialize isn't typical.